packages/0018-package-format.md

```diff
@@ -0,0 +1,150 @@
+# Project UmbGet Package format 📦
+
+Request for Contribution (RFC) 18 : _Package Format_
+
+## Code of conduct
+
+Please read and respect the [RFC Code of Conduct](https://github.com/umbraco/rfcs/blob/master/CODE_OF_CONDUCT.md)
+
+## Intended Audience
+
+The intended audience for this RFC is: Package developers and consumers
+
+## Summary
+
+Umbraco currently has two used package formats: 
+- The out of the box Umbraco ZIP package that can be generated from the backoffice
+- The NuGet package that is the standard in .NET
+
+This RFC's purpose is to suggest a way to bridge the gap and unify the two package formats while also looking closer at what a modern package format *should* offer.
+
+## Motivation
+
+A lot of package developers end up publishing their package as both a Zip and a NuGet package as both formats have unique advantages. However, as a developer, working with NuGet packages are highly preferred, as it has some very essential features - the main one being automatic dependency handling.
+
+We want to make it easier and simpler to create packages by only maintaining one format while getting the best features of both.
+
+## Detailed Design
+
+The high-level overview is that we want to remove the zip package format and move to a NuGet zip (.nupkg), and then make sure that that the benefits of the zip format is maintained:
+
+- Listing in the backoffice
+- Installing from the backoffice
+- Ability to install Umbraco content and schema
+- Ability to run code within the Umbraco context on install and uninstall
+
```

Comment:
I think creating a package from the backoffice is missing here.

Reply:
Agreed, will add it 🙂